lib/hammer_cli_csv/base.rb

```diff
@@ -12,7 +12,7 @@ class BaseCommand < HammerCLI::Apipie::Command
     option %w(--threads), 'THREAD_COUNT', 'Number of threads to hammer with',
            :default => 1, :hidden => true
     option %w(--export), :flag, 'Export current data instead of importing'
-    option %w(--file), 'FILE_NAME', 'CSV file (default to /dev/stdout with --csv-export, otherwise required)'
+    option %w(--file), 'FILE_NAME', 'CSV file (default to /dev/stdout with --export, otherwise required)'
```

Comment:
https://github.com/Katello/hammer-cli-csv/blob/master/lib/hammer_cli_csv/base.rb#L23

The old command is present and marked deprecated but I noticed that I hadn't updated this option's note.